examples/astro/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "astro",
+  "type": "module",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro check && astro build",
+    "preview": "astro preview",
+    "astro": "astro",
+    "postcss": "postcss src/main.css -o output.css"
+  },
+  "dependencies": {
+    "@astrojs/check": "^0.9.3",
+    "@astrojs/react": "^3.6.2",
+    "@juxio/css": "^0.7.6",
+    "@juxio/postcss": "^0.7.6",
```

```suggestion
    "@juxio/postcss": "workspace:*",
```

this applies for any @juxio/*